native/common/jp_gc.cpp

```diff
@@ -63,6 +87,11 @@ void JPGarbageCollection::triggered()
 
 JPGarbageCollection::JPGarbageCollection(JPContext *context)
 {
+#if defined(USE_PROC_INFO)
+	statm_fd = open("/proc/self/statm", O_RDONLY);
```

Comment:
you should also close this fh again.

Reply:
Done.  I also set the default for linux back to GLIBC when available.   The /proc/self/statm figures does not shrink when memory is releases so it is hard to properly set the lower bounds.